src/inverted-index.js

```diff
@@ -1,85 +1,89 @@
-'use strict';
-class Index {
 /**
- * Get Index method that indexes JSON file
- * @param data (key,value pair)
- * @return fileIndex
+ * Function that regularizes input
+ * @param {String} tokens
+ * @return {Array} newToken
  */
-    getIndex(data) {
-        let fileIndex = {};
-        for(let i=0;i<data.docs.length;i++) {
-            fileIndex[data.name] = [i];
-        }
-        return fileIndex;
+const tokenize = (tokens) => {
+  const newToken = tokens.replace(/[^a-z0-9]+/gi, ' ')
+    .trim().toLowerCase();
+  // send out the new token
+  return newToken;
```

Comment:
The variable `newToken` is redundant here. You're declaring it just to return the value.

What do you think about returning said value directly?